client/src/components/personsList/personsListComponent.js

```diff
@@ -0,0 +1,19 @@
+const UsersListController = require('./personsListController.js');
+
+var personsListComponent = {
+	bindings: {
+		persons: '=?'
```

Comment:
Why it's optional ? Can this list component work without this parameter passed ?